.github/workflows/kafka-collector-config/config_good.hocon

```diff
@@ -0,0 +1,85 @@
+collector {
```

Comment:
There's nothing wrong with this config, but he new minimal configs are allowed to be much shorter, see https://github.com/snowplow/stream-collector/blob/2.7.0/examples/config.kafka.minimal.hocon

Reply:
Sure, I'll reduce the config as much as possible.

Reply:
Reduced it to the bare minimum.